packages/contractkit/src/wrappers/Attestations.ts

```diff
@@ -223,23 +213,15 @@ export class AttestationsWrapper extends BaseWrapper<Attestations> {
   async getVerifiedStatus(
     identifier: string,
     account: Address,
-    numAttestationsRequired: number = DEFAULT_NUM_ATTESTATIONS_REQUIRED,
-    attestationThreshold: number = DEFAULT_ATTESTATION_THRESHOLD
-  ): Promise<AttestationsStatus> {
+    numAttestationsRequired?: number,
+    attestationThreshold?: number
+  ) {
     const attestationStats = await this.getAttestationStat(identifier, account)
-    const numAttestationsRemaining = numAttestationsRequired - attestationStats.completed
```

Comment:
This stuff was moved to `utils/src/attestations.ts` to make it easier to use in certain parts of the wallet